images/router/haproxy/conf/haproxy-config.template

```diff
@@ -164,8 +189,10 @@ backend be_edge_http_{{$cfgIdx}}
 backend be_tcp_{{$cfgIdx}}
   balance leastconn
   timeout check 5000ms
+  stick-table type ip size 1m expire 5m{{ if (len $.PeerEndpoints) gt 0 }} peers openshift_peers {{ end }}
```

Comment:
The 5 minutes for stick tables is the max time that an entry will stay without a refresh.  Any activity on the connection should refresh that 5 minute lease on the entry. Five minutes seems like an awful long time to not have any activity (to me) unless, perhaps, some large edi data processing that must pre-process on the client side after establishing the server connection.  Not sure that is really an issue though, I can think of multiple ways to mitigate that risk.

That said, since we have a max size on the table the oldest could be aged out if we had a higher expiry.  I'm not opposed to changing it.